lib/s3_meta_sync/syncer.rb

```diff
@@ -132,12 +132,17 @@ def upload_file(source, path, destination)
       log "Uploading #{path}"
       content = File.read("#{source}/#{path}")
       content = Zip.zip(content) if @config[:zip] && path != META_FILE
-      s3.put_object(
+
+      object = {
         bucket: @bucket,
         body: content,
         key: "#{destination}/#{path}",
-        acl: 'public-read'
-      )
+        acl: @config.fetch(:upload_acl, 'public-read')
+      }
+
+      object[:server_side_encryption] = @config[:server_side_encryption] if @config[:server_side_encryption]
```

Comment:
Because I don't want to option to be part of the hash if it isn't in the config. If you don't want SSE, then you don't pass the option along.